frontend/ui/widget/menu.lua

```diff
@@ -650,7 +649,7 @@ function Menu:updateItems(select_number)
                 show_parent = self.show_parent,
                 state = self.item_table[i].state,
                 state_size = self.state_size or {},
-                text = self.item_table[i].text,
+                text = self.item_table[i].text .. ((self.item_table[i].sub_item_table ~= nil) and " \226\150\185" or ""),
```

Comment:
can we create a local variable above and wrap this logic in a plain old if block? One liner like this is easier to write but less efficient to run and harder to read/maintain ;)